src/pages/Scanner/Scanner.module.css

```
@@ -0,0 +1,38 @@
+.main {
+  min-height: 100vh;
+  position: relative;
+  overflow: hidden;
+  background-color: var(--mantine-color-dark-9);
+  background-image: 
+    linear-gradient(rgba(255, 61, 0, 0.1) 1px, transparent 1px),
```

Stylelint (17.7.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)